DA-338: Add feedback box to dashboard

stgrue opened the pull request:
The dashboard now contains an alert box reminding users they can give
feedback to the developers.

judembo commented:
Hmm I think it might look better at the bottom of the page?
Also, i don't really see why this welcome "Hello <user>..." text needs to be highlighted in green. It used to be an alert box that would disappear after a few seconds and then it made sense, but now that it stays there the whole time I think we might as well remove the highlighting?

Commit:
DA-338: Add feedbax board to dashboard

The dashboard now contains an alert box at the bottom reminding users they can
give feedback to the developers.

stgrue commented:
Updated. The alert box is now at the bottom. I've left the green outline of the welcome message though, because it has a "login confirmed" feel to it (in my opinion).
